specification/security/resource-manager/Microsoft.Security/preview/2021-01-01-preview/examples/secureScores/ListSecureScoreControls_subscription_example.json

```diff
@@ -0,0 +1,218 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/secureScores/ascScore/secureScoreControls/a9909064-42b4-4d34-8143-275477afe18b",
```

Comment:
this is the example file for the top-level secureScoreControls type - so please remove mentions of /secureScores from the id and type

Reply:
> this is the example file for the top-level secureScoreControls type - so please remove mentions of /secureScores from the id and type

A secureScoreControls type is an extension of secure score (calculation) type.
There is no meaning of secure score control without secure score type.

@varunkumta  What is the best practice here ?
The motivation here was to list all subscription's security controls (And thus, the user can aggregate by their secure score)

Reply:
as per your manifest and your swagger there are 2 types, one is secureScores/secureScoreControls and the other is the top-level secureScoreControls .. even though the schema for the two is the same - they are technically different as they live at different scopes - their apis therefore must be consistent to the type i.e. top-level vs child

Reply:
The purpose of the top-level get is to expose all the child resources across all secureScores in a subscription.